### PR TITLE
Updated setup instructions in readme, backfilled changelog

### PR DESCRIPTION
**Pull request by PeterGallagherBJSS**
## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +127,11 @@ There are `make` commands that alias some of this functionality:
 ### Running tests
 #### Sandbox Tests
 
+To start the sandbox, run the install command if not ran already
```
Comment (richardseacome1-nhs) [inline README.md:130]: Wording slightly confusing here, maybe remove 'To start the sandbox', as technically, we're running 'make install'

File: README.md [hunk 2]
```diff
@@ -19,18 +19,89 @@ This code is dual licensed under the MIT license and the OGL (Open Government Li
...
 
-### Requirements
-* make
-* nodejs v12 or v14
-* npm/yarn
-* [poetry](https://github.com/python-poetry/poetry)
+## Setup
+
+N.B. that some functioanlity requires environment variables to be set. Some of these are described lower down in the readme, whilst others can be found in [the environment variables section of this confluence page](https://nhsd-confluence.digital.nhs.uk/display/SPINE/Personal+Demographics+Service+api+setup)
+
```
Comment (richardseacome1-nhs) [inline README.md:26]: I think it's worth mentioning the APIGEE_USERNAME and APIGEE_PASSWORD that will need to be added to .bashrc after the env_variables.txt file has been copied over to .bashrc. Obviously, this applies for first time setup and if a dev is handling other repos, their implementation will need a little more thought than copying over the whole file into .bashrc.

**Review by richardseacome1-nhs — COMMENTED** [review]
Couple of minor points to add/change

**Review by richardseacome1-nhs — CHANGES_REQUESTED** [review]
I think you also need to make sure all tests passing, so need to investigate failing test.
image

File: README.md (hunk 2 above)
Comment (PeterGallagherBJSS) [reply README.md:26]: The README does mention APIGEE_USERNAME and APIGEE_PASSWORD at the bottom of the page in the API Proxy & Sandbox Service section. I believe it's only needed for deployment, not to get the sandbox working locally so I think it's ok in that section. Let me know if you disagree.

**Review by richardseacome1-nhs — APPROVED** [review]

**Pull request merged**